### DISCUSSION TITLE
[FeatureRequest] Disable Auto Sorting on Pie Chart

### DISCUSSION THREAD

**Discussion by Vollbrecht** [grafana/grafana#49689] [Dashboards, Panels & Visualizations]
Currently the Pie chart panel auto sorts all values in a clock wise manner, in a descending order. If one uses multiple Pie charts the it starts to look out of place.
Here is an example :
![Screenshot from 2022-05-26 13-57-24](https://user-images.githubusercontent.com/12041081/170484523-cc8b147f-073b-4f0c-9589-2011f0997fab.png)
 It would be nice to have a fixed ordering, that doesn't sort automatically. This would also eliminate the extra need for an tooltip ordering in the Pie chart.

**Comment by jehutywong** [comment]
+1

**Comment by cholzer79** [comment]
PUSH!!!
The default sort behaviour is SO DISTRACTING.
Even worse is that the presentation eventually "flips" as soon as the lower grows and exceeds the previously higher value!!!!

![Screenshot 2023-09-06 101508](https://github.com/grafana/grafana/assets/64579423/3d67842f-ac9c-4e9a-8797-27b7bf6d4ef7)

btw. Tooltip ordering is also broken and does not affect the order values are presented in the tooltip.

**Comment by alkanen** [comment]
I completely agree.  The sorting is perfectly fine for single unrelated pie charts, but when you use Repeat options to get the same view of multiple objects and they all seemingly randomly sort the wedges it's borderline impossible to compare them to each other.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82972. Feel free to continue the discussion around this there. Thank you!
